src/Raven.Server/Documents/PeriodicBackup/Restore/Sharding/ShardedRestoreOrchestrationTask.cs

```diff
@@ -109,6 +112,7 @@ private void InitializeShardingConfiguration()
                     {
                         shardRestoreSetting.NodeTag
                     },
+                    ClusterTransactionIdBase64 = clusterTransactionIdBase64,
```

Comment:
what about the orchestrator topology? let's make sure it is also identical (probably not only a restore thing)